data/json/recipes/recipe_weapon.json

```diff
@@ -6621,6 +6621,54 @@
         [ "UPS_off", 2]
       ]
     ]
+},
+{
+   "type": "recipe",
+  "result": "Abzats",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics"
+  "skills_required": ["gun", 4],
```

Comment:
Github complains about this line, because the line above lacks a comma at the end.